test/Sentry.Maui.Tests/MauiEventsBinderTests.GestureRecognizers.cs

```diff
@@ -0,0 +1,145 @@
+using Sentry.Maui.Internal;
+
+namespace Sentry.Maui.Tests;
+
+public partial class MauiEventsBinderTests
+{
+#if VISUAL_RUNNER
+    [Fact]
+#else
+    [Fact(Skip = "Flaky Test in XHarness")]
```

Comment:
These are unit tests right, so they don't get run by the visual runner? They just get run by XHarness along with the other device tests? In which case, whether we skip these or not probably isn't dependent on whether the visual runner is enabled or not so much as whether we're running the tests on a device or not.

In that case, it'd be better if the directive was `#if !ANDROID && !IOS`... to make it clear this has nothing to do with the visual runner.

Reply:
They get run by the visual runner only.  The gesture recognizers are crazy flaky in the headless setup.  They run perfectly in the visual runner and full scenarios.